### PR TITLE
Added basic styles for the animation

### PR DESCRIPTION
**Pull request by jesiggar**
To test the animation just add 'active' class to '.rajoyirize-content-wrapper' div.

### CODE REVIEW

File: styles/main.scss [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+#rajoyirize-container {
+    font-family: Arial;
+    font-size: 16px;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    overflow: hidden;
+    
+    &.hidden {
```
Comment (aPlana) [inline styles/main.scss]: The idea of `hidden` was to be reused by the other elements. However, as we have a container wrapping everything, maybe this is fine.

File: styles/main.scss [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+        display: none !important;
+    }
+    
+    .rajoyirize-content-wrapper {
```
Comment (aPlana) [inline styles/main.scss:14]: I'm sure you already tried that but... is this container really necessary? If you apply the same styling to the element `#rajoyirize-container`, which is already a container, is not enough?

File: styles/main.scss [hunk 3]
```diff
@@ -1,9 +1,44 @@
...
+        img {
+            float: left;
+        }
+        
+        p {
+            float:left;
+            display: block;
+            width: 100%;
+            margin: 0;
+            bottom: 0;
```
Comment (aPlana) [inline styles/main.scss:36]: Is this really necessary? I don't think it has any effect, without changing the way the element is positioned.

File: styles/main.scss [hunk 4]
```diff
@@ -1,9 +1,44 @@
...
+            padding: 20px;
+            background: #000;
```
Comment (aPlana) [inline styles/main.scss:38]: Have you tried this as `background: rgba(0, 0, 0, 0.8)`? (so you don't need `opacity: 0.8`)

File: styles/main.scss [hunk 5]
```diff
@@ -1,9 +1,44 @@
...
+            opacity: 1;
+        }
+        
+        img {
+            float: left;
+        }
+        
+        p {
+            float:left;
+            display: block;
```
Comment (aPlana) [inline styles/main.scss]: Isn't a `p`element a block, by default?

**Comment by aPlana** [comment]
Cool (initial) animation! 👍

File: styles/main.scss (hunk 2 above)
Comment (jesiggar) [inline styles/main.scss:14]: That content wrapper allows to animate all elements (image and quotes) as a whole element, having the animation area delimited by #rajoyirize-container.

File: styles/main.scss (hunk 1 above)
Comment (jesiggar) [inline styles/main.scss]: The idea is not to overwrite a "hidden" class the page the plugin is inserted in might already have. But we should add ".hidden" to this declaration.

`&.hidden, .hidden {
        display: none !important;
    }`

File: styles/main.scss (hunk 5 above)
Comment (jesiggar) [inline styles/main.scss]: True! That can go away :)

File: styles/main.scss (hunk 3 above)
Comment (jesiggar) [inline styles/main.scss:36]: Need to try this, could be something I forgot to clean.

File: styles/main.scss (hunk 4 above)
Comment (jesiggar) [inline styles/main.scss:38]: Yeah. Personally, I prefer to keep both properties separate, makes it easier to spot that you are playing with the opacity.

**Comment by aPlana** [comment]
Yeah! GJ. Merging.

**Pull request merged**